src/main/java/dk/cngroup/lentils/entity/User.java

```diff
@@ -0,0 +1,77 @@
+package dk.cngroup.lentils.entity;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.persistence.Column;
+import javax.persistence.OneToMany;
+import javax.persistence.CascadeType;
+import javax.persistence.FetchType;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import java.util.Set;
+
+@Entity
+@Table(name = "user")
+public class User {
+
+    public User(final User user) {
```

Comment:
Je tento konstruktor někde použitý? Jestli ne, tak bych ho vyhodil. Podle mě smysl nemá.

Reply:
bez nej se nedá přihlásit :)